packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -155,15 +157,23 @@ class FrameFlameChart extends CoreElement {
 
   // TODO(kenzie): re-assess this drawing logic.
   void _drawFlameChartItem(TimelineEvent event, int left, int width, int top) {
-    final CoreElement item = div(text: event.name, c: 'flame-chart-item');
-    item.element.style.background = event.isCpuEvent
+    final item = Element.div()..className = 'flame-chart-item';
+    final wrapper = Element.div()..className = 'flame-chart-item-label-wrapper';
```

Comment:
nit: maybe rename this to labelWrapper? Initially I thought that `wrapper` would wrap `item`, so `item.append(wrapper)` threw me off.

Reply:
Done. Also added a comment on why maxSize is used.